docs/2.5. APIs - Query Parameters.md

```diff
@@ -12,60 +12,65 @@ Query APIs SHOULD support pagination of their API resources where the 'paged' tr
 
 Query API clients MUST detect whether pagination is being used by examining the HTTP response headers for 'X-Paging-Limit' which MUST be returned in all cases where pagination is in use.
 
-The following implementation notes should be observed:
+The registry which backs the Query API SHOULD maintain a 'creation' and 'update' timestamp alongside each registered resource. These values SHOULD NOT be returned to API clients in the response body, but will be made available via headers and used as pagination cursors.
 
-* The registry which backs the Query API is expected to maintain a 'creation' and 'update' timestamp alongside each registered resource. These values should not be returned to API clients in the response body, but will be made available via headers and used as pagination cursors.
+To ensure that pagination does not result in resources being skipped, there SHOULD NOT be duplicate creation or update timestamps stored against resources of the same type (Node, Source, Flow etc). It is RECOMMENDED that these timestamps are stored with nanosecond resolution using a TAI timebase, which will allow clients to navigate collections based on a common understanding of time.
 
-* In order to ensure that pagination does not result in resources being skipped, it is important that there are no duplicate creation or update timestamps stored against resources of the same type (Node, Source, Flow etc). It is suggested (although not mandated) that these timestamps are stored with nanosecond resolution using a TAI timebase, which will allow clients to navigate collections based on a common understanding of time.
+The choice to page based on 'creation' or 'update' timestamp SHOULD depend on the client's intended use of the data:
```

Comment:
Is this stating a recommendation?

Reply:
On reflection this can go back to being a suggestion.

Reply:
```suggestion
The choice to page based on 'creation' or 'update' timestamp depends on the client's intended use of the data:
```

Reply:
Done